src/Router/types.ts

```diff
@@ -0,0 +1,24 @@
+import { ReactNode } from 'react';
+
+export interface RouterNavigateFunction {
+    (to: string): void;
+}
+
+export interface IRouter {
+    navigate: (path: string) => void;
+}
+
+export enum RouteName {
+    LOGIN = '/login',
+    REGISTRATION = '/register',
+    LIBRARY = '/',
+    VIDEO = '/video'
+}
+
+export type RouterContextType = {
```

Comment:
There is a problem of type definition in this fille. It explain type for 2 levels. O would llike suggest define types forr React context near of this context definitioon

Reply:
@Rastopyr 
So better to don't create the `types.js` files? 
Better to declare types in related files? Like `IRouter` in RouterSerives file and `RouterContextType` in RouterContext?

Reply:
If you can't create it - avoid creation of `types.ts`. In some scenarios we will need it, but not in this case.